### ISSUE TITLE
TypeError: Got both 'size' and 'fontsize', which are aliases of one another

### ISSUE THREAD

**Issue by marcnol** [marcnol/pyHiM#24]
alignment3D raises the following issue (see below).
It only appears in python 3.9 and not in python 3.7. But presumably related to a new version of Matplotlib raising a TypeError that was not being raised before?

`  File "/home/marcnol/Repositories/pyHiM/src/imageProcessing/alignImages3D.py", line 151, in alignFiducials3Dfile
    return _alignFiducials3Dfile(fileName2Process,
  File "/home/marcnol/Repositories/pyHiM/src/imageProcessing/alignImages3D.py", line 410, in _alignFiducials3Dfile
    fig2 = plots3DshiftMatrices(shiftMatrices, fontsize=8)
  File "/home/marcnol/Repositories/pyHiM/src/imageProcessing/imageProcessing.py", line 2125, in plots3DshiftMatrices
    imageShowWithValuesSingle(axis, x, title, fontsize, cbar_kw, valfmt=valfmt, cmap="YlGn")  # YlGnBu
  File "/home/marcnol/Repositories/pyHiM/src/imageProcessing/imageProcessing.py", line 1917, in imageShowWithValuesSingle
    _ = annotate_heatmap(im, valfmt=valfmt, size=20, threshold=None, textcolors=("black", "white"), fontsize=fontsize)
  File "/home/marcnol/Repositories/pyHiM/src/imageProcessing/imageProcessing.py", line 2281, in annotate_heatmap
    text = im.axes.text(j, i, valfmt(data[i, j], None), **kw)
  File "/home/marcnol/anaconda3/envs/pyHiM39/lib/python3.9/site-packages/matplotlib/axes/_axes.py", line 659, in text
    t = mtext.Text(x, y, text=s, **effective_kwargs)
  File "/home/marcnol/anaconda3/envs/pyHiM39/lib/python3.9/site-packages/matplotlib/text.py", line 160, in __init__
    self.update(kwargs)
  File "/home/marcnol/anaconda3/envs/pyHiM39/lib/python3.9/site-packages/matplotlib/text.py", line 164, in update
    kwargs = cbook.normalize_kwargs(kwargs, Text)
  File "/home/marcnol/anaconda3/envs/pyHiM39/lib/python3.9/site-packages/matplotlib/cbook/__init__.py", line 1739, in normalize_kwargs
    raise TypeError(f"Got both {canonical_to_seen[canonical]!r} and "
TypeError: Got both 'size' and 'fontsize', which are aliases of one another`

**Comment by marcnol** [comment]
I have attempted to fix this by commenting the fontsize argument in annotate_heatmap as follows:

```python
    _ = annotate_heatmap(im, valfmt=valfmt, size=20, threshold=None, textcolors=("black", "white")) #, fontsize=fontsize
```
this fix is in commit `4077982b1b885632d49386e52cb2506d0cba254b`

**Comment by marcnol** [comment]
The bug does not raise anymore with this fix. It remains to be checked if the output figure has the right font sizes...!
